README.md

```diff
@@ -1,7 +1,9 @@
 # go-ecs-deploy [![Build Status](https://travis-ci.org/vend/go-ecs-deploy.svg)](https://travis-ci.org/vend/go-ecs-deploy)
-deploy a hosted docker container to an existing ecs cluster
+~~deploy a hosted docker container to an existing ecs cluster~~
 
-Allows deployment of ECS microservices straight from the command line!
+~~Allows deployment of ECS microservices straight from the command line!~~
+
+Note that **go-ecs-deploy** has been deprecated in favor of using [catapult](https://github.com/vend/catapult), as is done in the standard Travis CI/CD [deploy script](https://github.com/vend/ci-scripts/blob/master/travis/deploy.sh#L53).
```

Comment:
This is a public repo, so it may not make sense to mention an internal tool. (I don't think there are too many users of this public repo, but there may be a few.)

Reply:
Ah, I missed that this was a public repo. So it's only "deprecated" in that Vend itself is no longer using it. Which should be noted elsewhere. I'll just close this PR then. Thanks!